Source/Core/DolphinQt/Config/LogWidget.h

```diff
@@ -46,5 +46,7 @@ class LogWidget final : public QDockWidget, LogListener
   QPushButton* m_log_clear;
   QPlainTextEdit* m_log_text;
 
+  QTimer* m_timer;
```

Comment:
Kinda funny that you removed this in the 2nd commit only to re-add it back in the last one ;)

Reply:
There's more to come! I'll be re-adding a mutex too :)

I think I'll rewrite history after testing is done and streamline such changes.

Reply:
Oh, good to know; guess I approved early then